lib/avo/fields/files_field.rb

```diff
@@ -37,6 +40,14 @@ def fill_field(model, key, value, params)
 
         model
       end
+
+      def viewer_component
+        "Avo::Fields::Common::Files::ViewType::#{view_type.to_s.capitalize}Component".constantize
+      end
+
+      def view_type
```

Comment:
I think I'd like to change this implementation. We should not rely on the params in a file field to get the view type. If we need it somewhere we should inject it to the method as a dependency from a context where we do have the `params` (aka the view layer).

Reply:
Agree, changed on last PR